pkg/sources/github/repo.go

```diff
@@ -225,8 +230,52 @@ func (s *Source) getReposByOrg(ctx context.Context, org string) error {
 	})
 }
 
-func (s *Source) orgListReposWrapper(ctx context.Context, org string, opts repoListOptions) ([]*github.Repository, *github.Response, error) {
-	return s.apiClient.Repositories.ListByOrg(ctx, org, &opts.(*orgListOptions).RepositoryListByOrgOptions)
+// userType indicates whether an account belongs to a person or organization.
+//
+// See:
+// - https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts
+// - https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-a-user
+type userType int
+
+const (
+	// Default invalid state.
+	unknown userType = iota
+	// The account is a person (https://docs.github.com/en/rest/users/users).
+	user
+	// The account is an organization (https://docs.github.com/en/rest/orgs/orgs).
+	organization
+)
+
+func (s *Source) getReposByOrgOrUser(ctx context.Context, name string) (error, userType) {
```

Comment:
The order of return values probably should be swapped. I'm expecting the linter to complain.